### PR TITLE
Register new package ErrorfreeArithmetic.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JeffreySarnoff/ErrorfreeArithmetic.jl](https://github.com/JeffreySarnoff/ErrorfreeArithmetic.jl)
Release: [v0.0.1](https://github.com/JeffreySarnoff/ErrorfreeArithmetic.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/JeffreySarnoff/ErrorfreeArithmetic.jl/builds/219889222)
cc: @JeffreySarnoff

### CODE REVIEW

**Comment by JeffreySarnoff** [comment]
that's better (problem was either config file typo, or something wanted Compat -- earlier version used)

**Comment by JeffreySarnoff** [comment]
afaik, good
(I had to rename appveyor.yml for this to report itself in a clean state)

File: ErrorfreeArithmetic/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5.1
+Compat 0.18.0
```
Comment (tkelman) [inline ErrorfreeArithmetic/versions/0.0.1/requires]: This also isn't using Compat at all. And does it depend on a bug that was fixed between 0.5.0 and 0.5.1?

**Comment by JeffreySarnoff** [comment]
fixed

**Pull request merged**